src/frontend/apps/lti_site/components/Transcripts/index.spec.tsx

```diff
@@ -1,12 +1,13 @@
-import { fireEvent, screen } from '@testing-library/react';
```

Comment:
Your commit message is not descriptive enough. Use the real component name, why it needs styling ? Why it needs to be refreshed ?